build.sbt

```diff
@@ -209,26 +209,15 @@ lazy val scalameta = crossProject
 lazy val scalametaJVM = scalameta.jvm
 lazy val scalametaJS = scalameta.js
 
-lazy val scalahost = project
-  .in(file("scalahost/core"))
-  .settings(
-    moduleName := "scalahost",
-    description := "Scala.meta semantic API integration for Scala 2.x (scalac).",
-    publishableSettings,
-    hasLargeIntegrationTests,
-    isFullCrossVersion,
-    libraryDependencies += "org.scala-lang" % "scala-compiler" % scalaVersion.value
-  )
-  .dependsOn(scalametaJVM)
-
 lazy val scalahostNsc = project
   .in(file("scalahost/nsc"))
   .settings(
-    moduleName := "scalahost-nsc",
```

Comment:
Let's see what happens to paradise in the future. It may very well be that we manage to integrate it directly into scalac. So far, both paradise 3.x and scalahost are sufficiently esoteric to not worry about this imo.